src/audio/external.c

```diff
@@ -1824,7 +1824,7 @@ void func_800C5CB8(void) {
 }
 
 void func_800C5D04(u8 playerId) {
-    if ((gPlayers[playerId].unk_044 & 0x20) == 0x20) {
+    if ((gPlayers[playerId].kartProps & PRESS_A) == PRESS_A) {
```

Comment:
MOVE_FORWARDS?

Reply:
Renamed to `THROTTLE` to describe Kart state. I don't believe it requires forward movement.